hack/testing-olm/assertions

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/utils"
```

Comment:
this looks very similar (duplicated) to the one in the olm-upgrade... is there a way we could have a common source for them?